lib/testing_api_onlyoffice_com/test_instance/main_page/document_builder_api/document_builder_introduction.rb

```diff
@@ -0,0 +1,40 @@
+require 'httparty'
+require 'onlyoffice_file_helper'
+require_relative 'left_side_navigation'
+module TestingApiOnlyfficeCom
+  # https://user-images.githubusercontent.com/18173785/37905775-9964ebb6-3108-11e8-8f98-480cbb1c2906.png
+  # /docbuilder/basic
+  class DocumentBuilderIntroduction
+    include PageObject
+    include LeftSideNavigation
+
+    DEFAULT_BUILDER_FILE_NAME = 'SampleText.docx'.freeze
+
+    # actions
+    # introduction
+    link(:introduction, xpath: '//a[@class="selected"]')
+    link(:generate_document, xpath: '//*[@id="generateButton"]')
+
+    def initialize(instance)
+      @instance = instance
+      super(@instance.webdriver.driver)
+      wait_to_load
+    end
+
+    def wait_to_load
+      @instance.webdriver.wait_until { generate_document_element.visible? }
+    end
+
+    def builder_works?
```

Comment:
So this is a second one method with name `def builder_works?`?

Reply:
> So this is a second one method with name `def builder_works?`?

If builder is a same part of some pages, and it behavior is stable and independent, create new class `Builder` will be good idea. In this case, need to create 2 same methods `builder` for return the Builder object